src/app/workspace/results/results.component.html

```diff
@@ -9,3 +10,15 @@
         </as-split-area>
     </as-split>
 </div>
+
+<!-- In case of 0 result -->
+<div class="no-results" *ngIf="!loading && numberOfAllResults === 0">
+    <p>Your search <span *ngIf="searchMode === 'fulltext'">- <strong> {{searchQuery}}</strong> -</span> did not match any documents.</p>
```

Comment:
this message should be centered in the middle of the page instead of being displayed on the left side

Reply:
Done.